lib/node_modules/@stdlib/blas/base/snrm2/README.md

```diff
@@ -160,6 +160,129 @@ console.log( out );
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
```

Comment:
@aman-095 You'll want to update the docs above to indicate that `strides <= 0` are now permitted. Should probably double check that similar language has been updated elsewhere.

Reply:
See L94.